vspace/src/main/java/edu/asu/diging/vspace/core/services/IModuleManager.java

```diff
@@ -24,4 +25,6 @@
     ISequence checkIfSequenceExists(String moduleId, String sequenceId);
     
     Page<IModule> findByNameOrDescription(Pageable requestedPage,String searchText);
+
+    void deleteModule(String moduleId) throws ModuleNotFoundException;
```

Comment:
Exceptions should be in the exception package inside `core`. Not sure how this one (and others) ended up in the web package, but it should be moved into the core package (it's thrown from within service package after all). Also, please move all other exceptions that are not specifically only used in the web package.